apps/timer/app.js

```diff
@@ -193,7 +193,7 @@ module.exports = function (activity) {
       afterFunc = args
     }
     sendCardToApp('ROKID.TIMER', {text: text})
-    activity.tts.stop()
+    activity.tts.stop().catch((err) => { logger.warn('stop tts err:', err) })
```

Comment:
I'd think it's an error level log.

Reply:
> I'd think it's an error level log.

This `tts.stop` error will not break following process, so I give it warning level.